portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +33,43 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    ArrayList<String> list = new ArrayList<String>();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Gson gson = new Gson();
+    String json = gson.toJson(list);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+}
```

Comment:
The formatting looks a bit weird. Have you tried the clang format tool? https://github.com/stekolla/sps-clang-format